agent/agent-tooling/build.gradle.kts

```diff
@@ -19,11 +19,9 @@ dependencies {
   implementation(project(":agent:agent-gc-monitor:gc-monitor-core"))
   implementation(project(":agent:agent-profiler:agent-diagnostics-api"))
   implementation(project(":agent:agent-profiler:agent-diagnostics"))
-  implementation(project(":agent:azure-monitor-exporter")) {
-    exclude("org.ow2.asm", "asm")
```

Comment:
@trask not sure about this exclude

Reply:
check `./gradlew :agent:agent-tooling:dependencies` to see if it's a transitive dependency